device/iot-device-client/pom.xml

```diff
@@ -97,11 +97,6 @@
             <artifactId>gson</artifactId>
             <version>2.8.0</version>
         </dependency>
-        <dependency>
-            <groupId>commons-codec</groupId>
-            <artifactId>commons-codec</artifactId>
-            <version>1.10</version>
-        </dependency>
```

Comment:
device client will get this commons dependency from our deps package, so no need to redeclare it here as well